common/config/rush/.pnpmfile.cjs

```diff
@@ -1,8 +1,5 @@
 function readPackage(pkg) {
-  if (
-    (pkg.name == "typedoc") &&
-    pkg.dependencies
-  ) {
+  if (pkg.name == "typedoc" && pkg.dependencies) {
```

Comment:
shouldn't we just update typedoc instead of doing this hack?

Reply:
Yes, Yash and I are about finished with the upgrade. Will publish bemetalsmith@5.0 today which should fix any build issues. You can try it out now if you'd like by using version 5.0.1-dev.2.